src/js/pouchdb-node.js

```diff
@@ -0,0 +1,169 @@
+/*
+
+    A node-specific extension of gpii.pouch that adds additional abilities, such as loading data from package-specific
+    paths.
+
+ */
+/* eslint-env node */
+"use strict";
+var fluid = require("infusion");
+var gpii  = fluid.registerNamespace("gpii");
+
+var fs   = require("fs");
+var os   = require("os");
+var path = require("path");
+require("../../");
+
+fluid.registerNamespace("gpii.pouch.node");
+
+/**
+ *
+ * Create our working directory if it doesn't already exist.
+ *
+ * @param that - The component itself.
+ *
+ */
+gpii.pouch.node.initDir = function (that) {
+    var fullPath = path.resolve(that.options.baseDir, that.options.dbOptions.name);
+
+    // fluid.log("pouch component '" , that.id, "' saving data to '", fullPath, "'...");
+
+    if (fs.existsSync(that.options.baseDir)) {
+        if (fs.existsSync(fullPath)) {
+            that.hasExistingDataDir = true;
```

Comment:
Yes, this is the means by which I guess at whether there is already data, to satisfy one of Cindy's requirements.  It is used in "loadDataIfNeeded":

https://github.com/the-t-in-rtf/gpii-pouchdb/blob/GPII-1897/src/js/pouchdb-node.js#L88

We should review the PouchDB and/or leveldown docs collectively, there must be a clearer and cleaner way to do this.